VideoLocker/src/main/java/org/edx/mobile/course/CourseService.java

```diff
@@ -7,9 +7,39 @@
 import retrofit.http.Query;
 
 public interface CourseService {
+    /**
+     * @param username (optional):
+     *                 The username of the specified user whose visible courses we
+     *                 want to see. The username is not required only if the API is
+     *                 requested by an Anonymous user.
+     * @param org      (optional):
```

Comment:
I was just copying everything, I'm not aware of which params we plan on using (Ideally, the server would generate the API bindings for us and they would match exactly)

But I'm happy to remove the ones that we don't yet use.